app/src/main/kotlin/co/netguru/baby/monitor/client/application/ApplicationModule.kt

```diff
@@ -43,23 +43,29 @@ class ApplicationModule {
 
     @AppScope
     @Provides
-    fun firebaseRepository(preferencesWrapper: FirebaseSharedPreferencesWrapper, context: Context) = FirebaseRepository(preferencesWrapper, context)
+    fun firebaseRepository(preferencesWrapper: FirebaseSharedPreferencesWrapper, context: Context) =
+        FirebaseRepository(preferencesWrapper, context)
 
     @AppScope
     @Provides
     fun applicationDatabse(context: Context) =
-            Room.databaseBuilder(
-                    context,
-                    AppDatabase::class.java,
-                    "baby-monitor-database"
-            )
-                .addMigrations(object : Migration(1, 2) {
+        Room.databaseBuilder(
+            context,
+            AppDatabase::class.java,
+            "baby-monitor-database"
+        )
+            .addMigrations(object : Migration(1, 2) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/feb0c51d1649c186c776faa50e562320ab9dc395/app/src/main/kotlin/co/netguru/baby/monitor/client/application/ApplicationModule.kt#L57"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>